CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## unreleased
+* Add more functions to SpatialRef implementation
```

Comment:
Looks like there's already an Unreleased heading, you can add this new item beneath it.

Reply:
Done.